### PR TITLE
[Defacepocalypse] De-deface payments

### PR DESCRIPTION
**Pull request by HugsDaniel**
#### What? Why?

~There's no issue but~ it's part of #2807

This also fixes https://github.com/openfoodfoundation/openfoodnetwork/issues/4134.

I imported and de-defaced views from spree/backend/app/views/spree/admin/payments and imported views that were not defaced. So now we have:

 - `app/views/spree/admin/payments/new.html.haml`
 - `app/views/spree/admin/payments/_form.html.haml`
 - `app/views/spree/admin/payments/index.html.haml`
 - `app/views/spree/admin/payments/_list.html.haml`
 - `app/views/spree/admin/payments/show.html.haml`
 - `app/views/spree/admin/payments/source_views/_gateway.html.haml`

No functional changes done, just bringing code from spree, converting to haml and applying the defaces.

#### What should we test?

Make sure this page works as before :

 - Order -> Payments

#### Release notes

Changelog Category: Changed
De-defaced order's new payment page, thus making OFN less entangled with Spree code.

### CODE REVIEW

**Comment by HugsDaniel** [comment]
@luisramos0 I imported various views from spree: 
 - payments/show.html.haml
 - payments/index.html.haml
 - payments/_list.html.haml
 - payments/source_views/_gateway.html.haml

Basically the ones we use here. `credit.html.erb` and `_bill_address_form.html.erb` remain in Spree as I don't see where we use them.

File: app/views/spree/admin/payments/source_views/_gateway.html.haml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+          \/
+          = payment.source.year
+        %dt
+          = Spree.t(:card_code)
+          \:
+        %dd= payment.source.verification_value
+    .omega.six.columns
+      %dl
+        %dt
+          = Spree.t(:maestro_or_solo_cards)
```
Comment (luisramos0) [inline app/views/spree/admin/payments/source_views/_gateway.html.haml]: you need to bring these translations from spree.

**Comment by mkllnk** [comment]
I just found that this fixes an S2 bug: #4134. @HugsDaniel Do you have an idea when you have time to fix the missing translations?

**Comment by HugsDaniel** [comment]
@mkllnk done

**Comment by lin-d-hop** [comment]
Successfully made a payment on an order from admin.
The field is now fully functional and editable and the payments are successfully processed using default dummy card.

Instance: FR staging
Browser: Chrome

Good to go :thumbsup:

**Pull request merged**